helper/subproc/subproc.go

```diff
@@ -45,12 +45,14 @@ func Print(format string, args ...any) {
 //
 // r should be a buffer containing output (typically combined stdin + stdout)
 // f should be an HCLogger Print method (e.g. log.Debug)
-func Log(r io.Reader, f func(msg string, args ...any)) {
+func Log(r io.Reader, f func(msg string, args ...any)) string {
 	scanner := bufio.NewScanner(r)
+	line := "unknown error"
 	for scanner.Scan() {
-		line := scanner.Text()
+		line = scanner.Text()
 		f("sub-process", "OUTPUT", line)
 	}
+	return line
```

Comment:
This effectively returns only the last chunk of the value being scanned.

Reply:
> This effectively returns only the last chunk of the value being scanned.

OK, I guess this will have to change to a string append (collect for all .Scan() ) and then return the unified string.

I'll check how to do that (limited gopher skills)

Reply:
does this seem OK? 
https://github.com/hashicorp/nomad/pull/18444/commits/e1435e3639609387527d8c8be94f559faa42f1a9

the last TrimSuffix is purely cosmetic. we could NOT do that and thus avoid having to include `"strings"`?

Reply:
changes incorporated in the latest commit